spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -670,7 +681,19 @@ function recvPacket(
         // for unordered channels we must set the receipt so it can be verified on the other side
         // this receipt does not contain any data, since the packet has not yet been processed
         // it's the sentinel success receipt: []byte{0x01}
-        abortTransactionUnless(provableStore.get(packetReceiptPath(packet.destPort, packet.destChannel, packet.sequence) === null))
+        packetRecepit = provableStore.get(packetReceiptPath(packet.destPort, packet.destChannel, packet.sequence))
+        if (packetRecepit != null) {
```

Comment:
I noticed that we are also emitting an event in this case in [ibc-go](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel/keeper/packet.go#L227-L233), so I added it here as well. I hope that's ok.